README.md

```diff
@@ -75,27 +78,26 @@ end
 
 ### Open your built-in microphone and speaker by name
 ```julia
-PortAudioStream("Built-in Microph", "Built-in Output") do stream
+PortAudioStream("default", "default") do stream
     write(stream, stream)
 end
 ```
 
 ### Record 10 seconds of audio and save to an ogg file
 
 ```julia
-julia> using PortAudio, SampledSignals, LibSndFile
+julia> using PortAudio, SampledSignals, LibSndFile, FileIO
 
-julia> stream = PortAudioStream("Built-in Microph", 2, 0)
-PortAudio.PortAudioStream{Float32,SIUnits.SIQuantity{Int64,0,0,-1,0,0,0,0,0,0}}
-  Samplerate: 48000 s⁻¹
-  Buffer Size: 4096 frames
-  2 channel source: "Built-in Microph"
+julia> stream = PortAudioStream("default", 2, 0)
```

Comment:
```suggestion
julia> stream = PortAudioStream("default", 1, 0)
```

FWIW, my iMac (despite its cost) has only a 1-channel microphone.
So I'd vote for a 1 channel example here to aid accessibility, but it's also OK as is...
(I kind of wish there were a "maximum number of channels" default, but another day.)

Reply:
Oh, I had overlooked the `maximum` option in the docs.  Thanks!